mypy/binder.py

```diff
@@ -103,19 +109,21 @@ def _get(self, key: Key, index: int=-1) -> Type:
                 return self.frames[i][key]
         return None
 
-    def push(self, node: Node, typ: Type) -> None:
-        if not node.literal:
+    def put_if_bindable(self, expr: Expression, typ: Type) -> None:
```

Comment:
Of course it is the binder's business, but the reader should know that it is possible.